rmgpy/molecule/resonance.py

```diff
@@ -129,6 +145,7 @@ def analyze_molecule(mol, save_order=False):
                 'is_aryl_radical': False,
                 'hasNitrogenVal5': False,
                 'hasLonePairs': False,
+                'is_bidentate':mol.is_bidentate(),
```

Comment:
There are some unaddressed minor suggestions of formatting (e.g., here ` 'is_bidentate': mol.is_bidentate(),` instead of ` 'is_bidentate':mol.is_bidentate(),`)
See if you agree with them to use consistent formatting

Reply:
Resolved